### PR TITLE
[pt-br] Adding brazilian portuguese translation of ingress controller page

### PR DESCRIPTION
**Pull request by gabrielalbino**
# Changes
Brazilian portuguese translation of the ingress controller page.
## Required steps
- [x] Content translated
- [x] Links to other pages with translations updated
## Related Issue
#13939

### CODE REVIEW

File: content/pt-br/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Controladores Ingress
+reviewers:
```
Comment (sftim) [inline content/pt-br/docs/concepts/services-networking/ingress-controllers.md:3]: nit: omit this line

File: content/pt-br/docs/concepts/services-networking/ingress-controllers.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## Utilizando múltiplos controladores Ingress
+
+Você pode ter [quantos controladores Ingress desejar](https://git.k8s.io/ingress-nginx/docs/user-guide/multiple-ingress.md#multiple-ingress-controllers) 
+em um cluster. Quando você cria um ingress, você deve especificar para cada ingress o [`ingress.class`](https://git.k8s.io/ingress-gce/docs/faq/README.md#how-do-i-run-multiple-ingress-controllers-in-the-same-cluster) 
+para indicar qual controlador ingress deve ser utilizado caso exista mais de um em seu cluster.
+
+Se você não definir uma classe, sua provedora de serviços talvez use um controlador ingress padrão.
+
+Idealmente, todos os controladores Ingress devem preencher essa informações, mas cada um dos diversos controladores ingress operam de maneira um pouco diferente.
```
Comment (sftim) [inline content/pt-br/docs/concepts/services-networking/ingress-controllers.md:65]: nit:
```suggestion
Idealmente, todos os controladores Ingress devem preencher essa informações, mas cada um dos diversos controladores Ingress operam de maneira um pouco diferente.
```

File: content/pt-br/docs/concepts/services-networking/ingress-controllers.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+* o [Traefik Kubernetes Ingress provider](https://doc.traefik.io/traefik/providers/kubernetes-ingress/) é um controlador Ingress para a proxy [Traefik](https://traefik.io/traefik/).
+* [Tyk Operator](https://github.com/TykTechnologies/tyk-operator) extende o Ingress com recursos customizados para trazer capacidades de gerenciamento de API para o Ingress. O Tyk Operator funciona com o painel de controle Tyk Gateway & Tyk Cloud
+* [Voyager](https://appscode.com/products/voyager) é um controlador Ingress para o
+  [HAProxy](https://www.haproxy.org/#desc).
+
+## Utilizando múltiplos controladores Ingress
+
+Você pode ter [quantos controladores Ingress desejar](https://git.k8s.io/ingress-nginx/docs/user-guide/multiple-ingress.md#multiple-ingress-controllers) 
+em um cluster. Quando você cria um ingress, você deve especificar para cada ingress o [`ingress.class`](https://git.k8s.io/ingress-gce/docs/faq/README.md#how-do-i-run-multiple-ingress-controllers-in-the-same-cluster)
```
Comment (sftim) [inline content/pt-br/docs/concepts/services-networking/ingress-controllers.md:60]: nit:
```suggestion
em um cluster. Quando você cria um Ingress, você deve especificar para cada Ingress o [`ingress.class`](https://git.k8s.io/ingress-gce/docs/faq/README.md#how-do-i-run-multiple-ingress-controllers-in-the-same-cluster)
```

**Review by sftim — COMMENTED** [review]
Looks like Portuguese to me (but I don't speak the language).

All the feedback I've added is both tiny and optional.

**Comment by devlware** [comment]
Hi @gabrielalbino, I agree with the suggestions from @sftim. If you could update those LGTM.

**Pull request closed without merging**